addons/project/models/project.py

```diff
@@ -60,6 +62,11 @@ def unlink(self):
                 stages = self.filtered(lambda x: x not in shared_stages)
         return super(ProjectTaskType, stages).unlink()
 
+    @api.depends('project_ids', 'project_ids.rating_active')
+    def _compute_project_rating_disabled_warning(self):
+        for r in self:
+            r.projects_rating_disabled = '\n'.join(map(lambda p: '- ' + str(p.name), r.project_ids._origin.filtered(lambda p: not p.rating_active)))
```

Comment:
why not 
```python3 
'\n'.join('- %s' % name for name in stage.project_ids._origin.filtered(lambda p: not p.rating_active).mapped('name')))
```
On the other hand, shouldn't we only take the first 5 or 10 projects ? To avoid an huge list displayed on the view.

Reply:
Done, I added a limit to 5 projects.